geo/line.go

```diff
@@ -0,0 +1,210 @@
+package geo
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/paulmach/orb"
+	"github.com/paulmach/orb/geo"
+	"github.com/paulmach/orb/planar"
+	"go.starlark.net/starlark"
+)
+
+func newLine(thread *starlark.Thread, _ *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (v starlark.Value, err error) {
+	var points *starlark.List
+	v = starlark.None
```

Comment:
I find this confusing, it looks like an unused variable. Took me a while to notice that it's a named return value. Would prefer to explicitly `return starlark.None, err` below, and delete this line.

Also, according to https://golang.org/doc/effective_go.html#named-results, named return values are initialized to their zero value. Do we know if `None` is a zero value? If so, all these assignments are unneeded.

Reply:
This comment is super helpful! I'll admit to feeling a little clever here, and am delighted to see my cleverness called out 😄. Will maek change you've suggested here.

`starlark.Value` is an interface type, so it's zero-value is `nil`, so if we want to return `starlark.None`, we need to explicitly set the return. I'm not sure, but I'd guess that the starlark package throws away the value if the returned error is non-nil, so this is probably all a nonstarter, I just want to be in the habit of thinking about the return of all starlark builtins being a valid starlark value.

Reply:
Ah, unfortunate. I think, generally I'm not a fan of the named return values thing, but that's a different conversion for a different time. As noted at https://tour.golang.org/basics/7, it really fits best for short obvious functions, and I mostly like it when it replaces some code that would require a var declaration anyway.